aten/src/ATen/test/quantized_test.cpp

```diff
@@ -131,14 +131,14 @@ TEST(TestQTensor, EmptyPerchannelQuantized) {
       {ch_axis},
       at::device(at::kCPU).dtype(kQUInt8));
   // Assigning to QTensor
-  auto* q_data = q.data<quint8>();
+  auto* q_data = static_cast<quint8*>(q.data_ptr());
```

Comment:
I think we can use `data_ptr<T>` here:
```suggestion
  auto* q_data = q.data_ptr<quint8>();
```